apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedDevelopCore.md

```diff
@@ -1,14 +1,14 @@
-[Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) is a collection of CSS classes and Sass mixins that give you access to Fabric's colors, animations, fonts, icons and grid.
+[Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) is a collection of CSS classes and Sass mixins that give you access to colors, animations, fonts, icons and grid.
```

Comment:
Can we add something here about how Fabric Core also implements elements of the Fluent design language (in CSS/Sass only) despite having a different name (assuming that's true)?